Makefile

```diff
@@ -75,9 +75,17 @@ build-unit-tests-image:
 run-unit-tests:
 	$(run_unit_tests_image) make test
 
+KUBERNETES_VERSION=1.23.1
+PATH_TO_KUBEBUILDER=/root/.local/share/kubebuilder-envtest/k8s
 .PHONY: test
 test: check-generated-manifests update
-	ginkgo -r -v -skipPackage tests
+ifeq (s390x, $(shell hack/get-arch.sh))
+	ginkgo -r -v -skipPackage envtest
+else
+	setup-envtest use -p path ${KUBERNETES_VERSION}
+	export KUBEBUILDER_ASSETS=${PATH_TO_KUBEBUILDER}/${KUBERNETES_VERSION}-linux-$(shell hack/get-arch.sh);\
```

Comment:
why do we want to maintain `PATH_TO_KUBEBUILDER`? can't we just take the output of `setup-envtest use ...`?

Reply:
done